client/components/domains/register-domain-step/index.jsx

```diff
@@ -220,7 +221,10 @@ const RegisterDomainStep = React.createClass( {
 	},
 
 	render: function() {
-		const queryObject = getQueryObject( this.props );
+		const queryObject = getQueryObject( this.props ),
+			placeholder = ( this.props.isSignupStep && abtest( 'signupDomainsHeadline' ) === 'updated' )
+				? this.props.translate( 'Enter a name or keyword' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Please enter a domain name or keyword.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).